language/stdlib/modules/lbr.move

```diff
@@ -44,7 +44,7 @@ module LBR {
             true,    // is_synthetic
             1000000, // scaling_factor = 10^6
             1000,    // fractional_part = 10^3
-            x"4C4252" // UTF8-encoded "LBR" as a hex string
+            b"LBR"
```

Comment:
Beautiful!

Reply:
@tzakian @sblackshear 
Any other spots to clean up?

Reply:
I searched for all the hex string literals but I suppose I might have missed something.

Reply:
No idea, just figured I'd ask since @tzakian had talked about having to go onto a website to get the encoding